src/to-be-checked.js

```diff
@@ -1,6 +1,26 @@
 import {matcherHint, printReceived} from 'jest-matcher-utils'
 import {checkHtmlElement} from './utils'
 
+/* eslint-disable valid-jsdoc */
```

Comment:
Why does this need to be disabled?

Reply:
It's disabled due to the linting errors (also for every other matcher):
```
Missing JSDoc @returns for function.eslint(valid-jsdoc)
Missing JSDoc for parameter 'element'
```

I chose to ignore the parameters and return comments in favor of describing the purpose of each matcher, along with examples.

---

I also didn't want to disable the rules for the entire file but only for those comments of code. Is there a better way to disable the jsdoc linting rule?